src/AddTodoForm.js

```diff
@@ -1,13 +1,26 @@
 import React from "react";
 
-function AddTodoForm () {
-    return(
-        <form>
-            <label htmlFor="todoTitle">Title</label>
-            <input id="todoTitle"/>
-            <button type="submit">Add</button>
-        </form>
-    )
+function AddTodoForm(props) {
+  function handleAddTodo(event) {
+    event.preventDefault();
+
+    const todoTitle = event.target.titel.value;
```

Comment:
Just a small typo here

```suggestion
    const todoTitle = event.target.title.value;
```